src/library/scala/collection/Seq.scala

```diff
@@ -766,7 +766,7 @@ trait SeqOps[+A, +CC[_], +C] extends Any
     */
   def indices: Range = Range(0, length)
 
-  override final def sizeCompare(_size: Int): Int = lengthCompare(_size)
+  override final def sizeCompare(otherSize: Int): Int = lengthCompare(otherSize)
```

Comment:
`otherSize` is the name of the param on the base definition

Reply:
There ought to be a lint for that. Or what's it called, Abide? No wait, ScalaCheck is the other thing, oh right, it's ScalaFix but without the Fix. There ought to be a ScalaFixless for that, to check that overrides align their param names. ("Does it matter?" he asked ~naively~ immaterially.)

Maybe call the linter `ScalaFox`. Is that too close to FIrefox? `ScalaFux`. That's better, because it's for when you's fuxed up.

Reply:
In intellij it's called an "inspection"